includes/modules/payment/paypalwpp.php

```diff
@@ -177,6 +177,8 @@ function __construct() {
 
       if (null === $this->sort_order) return false;
 
+      $this->enabled = (MODULE_PAYMENT_PAYPALWPP_STATUS === 'True' || (IS_ADMIN_FLAG === true && MODULE_PAYMENT_PAYPALWPP_STATUS === 'Retired'));
```

Comment:
$this->enabled is now inside the IS_ADMIN clause: this payment option will never show on the shopfront.

Reply:
True, that.  I'll get this updated later today.

Reply:
Sigh, the `paypaldp` has issues as well.  Both modules have some additional checks if enabled to set their title/description.

Reply:
@lat9 wouldn't just moving that $this->enabled check back to its original line be enough? (er, well, probably also restore the original `defined()` check as well).

I'm not seeing why it needed to be relocated.

Reply:
That's me over-thinking again; you're right.